test/integration/multinode_test.go

```diff
@@ -202,7 +202,7 @@ func validateCopyFileWithMultiNode(ctx context.Context, t *testing.T, profile st
 			if n.Name == n2.Name {
 				continue
 			}
-			fp := filepath.Join("/home/docker", fmt.Sprintf("cp-test_%s_%s.txt", n.Name, n2.Name))
+			fp := fmt.Sprintf("/home/docker/cp-test_%s_%s.txt", n.Name, n2.Name)
```

Comment:
I think `path.Join` will always do Linux style joins, but I don't see the harm is hardcoding it in those one case, since we're always looking inside of minikube.

Reply:
I actually didn't know that `path.Join` always used Linux joins, will update to use that.

Reply:
Updated